src/main/service/listManipulation/SjpPressListService.ts

```diff
@@ -75,8 +75,8 @@ export class SjpPressListService {
     private formatIndividualInformation(individualDetails) {
         return {
             name: listParseHelperService.createIndividualDetails(individualDetails),
-            dob: this.formatDateOfBirth(individualDetails),
-            age: individualDetails.age,
+            dob: individualDetails.dateOfBirth ? this.formatDateOfBirth(individualDetails) : '',
+            age: individualDetails.age ? individualDetails.age: 0,
```

Comment:
Will this render 0 in the frontend? If so I think this needs to be blank rather than 0